qa/tasks/cephfs/filesystem.py

```diff
@@ -607,7 +607,28 @@ def create(self):
 
         self.getinfo(refresh = True)
 
-        
+    def destroy(self):
```

Comment:
I don't think so, `delete_all_filesystems` is being used to cephfs_test_case.py and few other tests to delete all FSs on cluster _without_ having instances of `Fllesystem`  representing those FSs.

Reply:
`delete_all_filesystems` should be a static method anyway. It should take ` ctx` instead of `self`. There's no problem instantiating a throwaway `Filesystem` object for each file system during iteration.

Reply:
Makes sense. I'll make that change.

Reply:
@batrick Making `delete_all_filesystem()` a static method doesn't look like a good idea. Calling it as `Filesystem.delete_all_filesystems()` makes tests incompatible with vstart_runner.py and calling it as `LocalFilesystem.delete_all_filesystems()` would make it teuthology incompatible. I've written another method to pick `Filesystem` or `LocalFilesystem` automatically, but I think it's best to get rid of it and make `filesystem.py`'s `delete_all_filesystems()` instance method.

Reply:
Related thread - https://github.com/ceph/ceph/pull/32581#discussion_r455990575

Reply:
Made `delete_all_filesystems()` an instance method again.